client/src/partials/stock/dashboard/dashboard.js

```diff
@@ -12,6 +12,12 @@ function StockDashboardController($q, $http, DataService, Dates) {
   vm.purchaseorders = {};
   vm.limits = { consumption : 10, donations : 10 };
 
+  // loading indicators
+  vm.indicators = {
```

Comment:
Simple variable should not be in plural, except array but it is not a big problem